lib/licensed/sources/swift.rb

```diff
@@ -48,7 +53,7 @@ def pins
       end
 
       def dependency_path_for_url(url)
-        last_path_component = URI(url).path.split("/").last.sub(/\.git$/, "")
+        last_path_component = URI(url).path.split("/").last
```

Comment:
When Swift Package Manager clones Git repositories, it follows the same default behavior as the `git` client, where the resulting directory is named with the basename of the last path component. That's the behavior I intended to capture in this method.

Any objections to keeping this as-is, and making an explicit call to `.sub(/\.git$/, "")` in `#enumerate_dependencies`?

Reply:
Ah yeah I see what you mean, I had moved it around because it seemed unnecessary to do that operation twice, and it was still matching the intended behavior.  I don't have a strong feeling here and happy to change it back.